include/mbedtls/platform_util.h

```diff
@@ -54,6 +63,18 @@ extern "C" {
  *
  */
 void mbedtls_platform_zeroize( void *buf, size_t len );
+#else
+/* Windows implementation */
+#include <windows.h>
+#define MBEDTLS_PLATFORM_ZEROIZE_MACRO( buf, len )  \
+    RtlSecureZeroMemory( buf, len )
+#endif /* MBEDTLS_PLATFORM_ZEROIZE_ALT || !_MSC_VER */
```

Comment:
Minor remark: the test we close here checks _WIN32 rather than _MSC_VER

Reply:
Thanks. Will do that.